community_teams_list_page/community_teams_list_page.py

```diff
@@ -56,6 +64,11 @@ def generate_databag():
         if member["name"] == "":
             continue  # Sometimes blank names come up
         role = get_custom_field(member, "Role")
+        if "Community" in role:
```

Comment:
Could we do `role.startswith("Community")` instead? I think it would be less error prone.